generators/app/templates/src/main/java/package/service/kafka/deserializer/EntityDeserializer.java.ejs

```diff
@@ -31,51 +29,33 @@ import org.apache.kafka.common.serialization.Deserializer;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-<%_ if (type !== 'String') { _%>
 import java.io.IOException;
 
 import <%= packageName %>.domain.<%= entityClass %>;
-<%_ } else { _%>
-import java.io.UnsupportedEncodingException;
-<%_ } _%>
 
 public class <%= entityClass %>Deserializer implements Deserializer<Either<DeserializationError, <%= type %>>> {
 
     private final Logger log = LoggerFactory.getLogger(<%= entityClass %>Deserializer.class);
 
     private final ObjectMapper objectMapper;
 
-    private String encoding = "UTF8";
-
     public <%= entityClass %>Deserializer() {
         this.objectMapper =
             new ObjectMapper()
                 .registerModule(new ParameterNamesModule())
                 .registerModule(new Jdk8Module())
                 .registerModule(new JavaTimeModule())
                 .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
-                .setPropertyNamingStrategy(PropertyNamingStrategy.SNAKE_CASE)
-                .setSerializationInclusion(JsonInclude.Include.NON_EMPTY)
```

Comment:
To be confirmed. I would say to keep empty string as empty string and not mapping them to null ?

Reply:
Yep I agree with you!